src/score-result.js

```diff
@@ -0,0 +1,31 @@
+function scoreResult(qOne, qTwo, qThree, qFour, qFive) {
+    qOne = qOne.toLowerCase();
+    qTwo = qTwo.toLowerCase();
+    qThree = qThree.toLowerCase();
+    qFour = qFour.toLowerCase();
+    qFive = qFive.toLowerCase();
+
+    let score = 0;
+
+    if(qOne === 'bugs') {
+        score += 1;
```

Comment:
Quick tip: if you are adding in increments of 1 you can just put `score++` instead of `score += 1`

Reply:
Awesome thanks!